### PR TITLE
remove some duplicate code

### PR DESCRIPTION
**Pull request by psykzz**

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/freelook/cameranet.dm [hunk 1]
```diff
@@ -23,10 +23,7 @@ var/datum/cameranet/cameranet = new()
 // Returns the chunk in the x, y, z.
 // If there is no chunk, it creates a new chunk and returns that.
 /datum/cameranet/proc/getCameraChunk(x, y, z)
-	x &= ~(CHUNK_SIZE - 1)
-	y &= ~(CHUNK_SIZE - 1)
-	var/key = "[x],[y],[z]"
-	if(!chunks[key])
+	if(!chunkschunkGenerated(x, y, z))
```
Comment (phil235) [inline code/modules/mob/living/silicon/ai/freelook/cameranet.dm]: I think you mean  `!chunkGenerated(x, y, z)`

Comment (psykzz) [inline code/modules/mob/living/silicon/ai/freelook/cameranet.dm]: Fixed

**Comment by phil235** [comment]
:+1:

**Comment by MrStonedOne** [comment]
This PR caused #7671

Revert open at #7672

**Comment by phil235** [comment]
@psykzz 
The error came from `var/key = "[x],[y],[z]" on line 26` , it must be moved after `if(!chunkGenerated(x, y, z))` to get the correct coordinates for the chunk. I completely missed it.

**Comment by psykzz** [comment]
Wait their changed by `chunkGenerated` ?
I did not think that would happen.

I agree with reverting the PR, anything i do to fix this will be just as bad as the original method i think.

**Pull request merged**